doc/user/zebra.rst

```diff
@@ -58,6 +58,26 @@ Besides the common invocation options (:ref:`common-invocation-options`), the
 
 .. _interface-commands:
 
+Configuration Addresses behaviour
+=================================
+
+At startup, when running *Zebra*, the daemon will first discover the
```

Comment:
`s/, when running *Zebra*, the daemon/ *Zebra*`

Reply:
ok